lib/mshoplib/src/MShop/Service/Provider/Decorator/Supplier.php

```diff
@@ -179,7 +191,7 @@ public function setConfigFE( \Aimeos\MShop\Order\Item\Base\Service\Iface $orderS
 		if( ( $code = $attributes['supplier.code'] ) != '' )
 		{
 			// add short address as attribute for summary page / customer email
-			$attributes['supplier.address'] = $this->feConfig['supplier.code']['short'][$code];
+			$attributes['supplier.address'] = $this->feConfig['supplier.code'][$this->getConfigValue('supplier.addressformat')][$code];
```

Comment:
You should use a default value if it's not set like:
```php
$format = $this->getConfigValue( 'supplier.addressformat', 'short' );
$attributes['supplier.address'] = $this->feConfig['supplier.code'][$format][$code];
```

Reply:
Can i do it in oneline?
```
$attributes['supplier.address'] = $this->feConfig['supplier.code'][$this->getConfigValue('supplier.addressformat','short')][$code];
```

Reply:
Yes but the line will be very long and we try to keep the line length below 120 chars.

Reply:
ah, okay, understand, i'll change.

Reply:
done